src/networkapi.jl

```diff
@@ -574,6 +574,74 @@ function deficiency(ns, ig, lc)
    LG.nv(ig) - length(lc) - rank(matrix(FlintZZ,ns))
 end
 
+	
+
+"""
+	`subnetworkmapping` is an intermediary function
+	required in `subnetworks(network, linkage_classes)`
+"""
+# intermediary function for subnetwork of a particular linkage class
+function subnetworkmapping(linkageclass, r, reacmap, pars)
+   rxind = collect(Set(vcat([map(first, reacmap[j]) for j in 1:length(linkageclass)]...)))
```

Comment:
@yewalenikhil65 I still don't understand why this isn't
```julia
j in linkageclass
```
Shouldn't `j` be the complex index, not the position in the linkageclass vector?

Let me know about this and I'll handle changing if needed.